data/migrations/README

```diff
@@ -0,0 +1,14 @@
+# Alembic Docs
+
+[Alembic](https://alembic.sqlalchemy.org/en/latest/index.html) is a database migrations tool written by the author of [SQLAlchemy](http://www.sqlalchemy.org/) to be used every time that there's an update that will affect the current database.
+If you did an upgrade on the [Database Models](../../otter_welcome_buddy/database/models), you should add a new revision that will be applied when the bot goes online. In order to do that, follow this steps:
+
+1. Do the modifications on the Database Models.
+2. From your shell, run the following command:
+
+```bash
+alembic revision — autogenerate -m "<description_of_your_change>"
+```
+
+1. This will generate a new file under the [versions folder](versions/). Identify that file and open it.
```

Comment:
Can you explain in the main README how to setup the database and use Alembic? I'm getting confused how to run the project and I plan to start development after my vacations.

Reply:
You don't need to setup anything, everything is done automatically. This readme is more for development but I can link it in the README